External types d.ts is not published to the repo

### Expected behavior
SDK should be usable from TypeScript project

### Actual behavior
Build failed with
```
> lisk-core@3.0.0-beta.1 build /lisk-core
> tsc

node_modules/lisk-framework/dist-node/controller/channels/ipc_channel.d.ts:1:22 - error TS6053: File '/lisk-core/node_modules/lisk-framework/src/external_types/pm2-axon/index.d.ts' not found.

1 /// <reference path="../../../src/external_types/pm2-axon/index.d.ts" />
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/lisk-framework/dist-node/controller/channels/ipc_channel.d.ts:2:22 - error TS6053: File '/lisk-core/node_modules/lisk-framework/src/external_types/pm2-axon-rpc/index.d.ts' not found.

2 /// <reference path="../../../src/external_types/pm2-axon-rpc/index.d.ts" />

```

### Steps to reproduce
Install SDK from the registry and try to use from TypeScript

### Which version(s) does this affect? (Environment, OS, etc...)
5.0-

--- a/framework/package.json
+++ b/framework/package.json
@@ -32,7 +32,7 @@
 		"format": "prettier --write '**/*'",
 		"test": "jest --config=./test/unit/jest.config.js",
 		"test:coverage": "jest --config=./test/unit/jest.config.js --coverage=true --coverage-reporters=text",
-		"copy-static-files": "copyfiles -u 1 src/**/*.{yml} ./dist-node",
+		"copy-static-files": "copyfiles -u 1 src/**/*.d.ts ./dist-node",
 		"build": "tsc && npm run copy-static-files",
 		"test:unit": "jest --config=./test/unit/jest.config.js --coverage=true --coverage-reporters=json --verbose",
 		"test:integration": "jest --config=./test/integration/jest.config.js",

